Content.Shared/Revolutionary/SharedRevolutionarySystem.cs

```diff
@@ -14,13 +14,13 @@ public sealed class SharedRevolutionarySystem : EntitySystem
     public override void Initialize()
     {
         base.Initialize();
-        SubscribeLocalEvent<MindShieldComponent, ComponentInit>(MindShieldImplanted);
+        SubscribeLocalEvent<MindShieldComponent, MapInitEvent>(MindShieldImplanted);
```

Comment:
I didn't see the reason for this in the PR description, could you summarize why it should be `MapInitEvent`?

Reply:
ComponentInit is way too early in the initialize process. This is correct.